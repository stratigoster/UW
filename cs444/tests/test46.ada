PACKAGE foo IS
  type colors is (white,neon,black);
  type me is record
    null;
  end record;
  type pos is array (colors, white..black, me) of string;
--                                         ^^
-- `me' is not an enumeration 
END foo;
