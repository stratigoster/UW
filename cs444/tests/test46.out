***ERROR***:6:47:	Array indices must all be of enumeratable type, given ` me '

------
Compilation error
------
